### DISCUSSION TITLE
why do these modules have background colors even though I didn't define the background colors?

### DISCUSSION THREAD

**Discussion by julkefol** [Alexays/Waybar#2346] [Q&A]
![20230724_22h06m12s_grim](https://github.com/Alexays/Waybar/assets/75586433/c0e5a98d-9932-47d3-a4c4-a1038bbaa258)

My waybar config: [config](https://github.com/jcnafi/Dotfiles/blob/master/config/waybar/config) [style.css](https://github.com/jcnafi/Dotfiles/blob/master/config/waybar/style.css)

**Comment by MajorMuff** [comment]
Try setting
```
* {
  background: transparent;
}
```
